tools/rostest/nodes/advertisetest

```diff
@@ -135,19 +133,19 @@ class AdvertiseTest(unittest.TestCase):
                     if t_name in t_name_set:
                         t_name_set.remove(t_name)
                         topic = self.topics[t_name]
-                        assert_true(t_elapsed < topic['timeout'],
-                                    'Topic [%s] is advertised before timeout [%s] secs' % (topic['name'], topic['timeout']))
-                        assert_false(topic['negative'],
-                                     'Topic [%s] is not advertised' % (topic['name']))
+                        assert t_elapsed < topic['timeout'], \
+                            'Topic [%s] is advertised before timeout [%s] secs' % (topic['name'], topic['timeout'])
+                        assert not topic['negative'], \
+                            'Topic [%s] is not advertised' % (topic['name'])
```

Comment:
If this is asserting `not topic['negative']`, would an assertion failure mean the topic was unexpectedly advertised?

Reply:
Correct.